librisworks/scripts/merge-works.groovy

```diff
@@ -23,12 +22,17 @@ WorkStatus.values().each {
     new File(reportsDir, it.status).with { it.mkdirs() }
 }
 
-new File(System.getProperty('clusters')).splitEachLine(~/[\t ]+/) { cluster ->
-    def docs = Collections.synchronizedList([])
-    selectByIds(cluster) {
-        docs.add(new Doc(it))
-    }
+def changedBy = 'SEK'
+def generationProcess = 'https://libris.kb.se/sys/merge-works'
+
+def newWorks = []
+def idToUpdatedWorkData = [:]
+def instanceIdToWorkId = [:]
+def dataForReports = []
+def dataForMultiWorkReport = []
 
+def clusters = new File(System.getProperty('clusters')).collect {it.split(/[\t ]+/) as List }
```

Comment:
These are local to each script thread, since they're `def`:ed, right?

Reply:
These are used read-only inside selects (where threads are used) so should be safe.